release.json

```diff
@@ -23,146 +23,50 @@
-        "MACOS_BUILD_VERSION": "7.29.0-rc.1",
-        "WINDOWS_DDNPM_DRIVER": "release-signed",
-        "WINDOWS_DDNPM_VERSION": "0.98.2.git.86.53d1ee4",
-        "WINDOWS_DDNPM_SHASUM": "5d31cbf7aea921edd5ba34baf074e496749265a80468b65a034d3796558a909e"
-    },
-    "6.29.0-rc.3": {
-        "INTEGRATIONS_CORE_VERSION": "7.29.0-rc.2",
-        "OMNIBUS_SOFTWARE_VERSION": "7.29.0-rc.1",
-        "OMNIBUS_RUBY_VERSION": "7.29.0-rc.1",
-        "JMXFETCH_VERSION": "0.44.1",
-        "JMXFETCH_HASH": "fd369da4fd24d18dabd7b33abcaac825d386b9558e70f1c621d797faec2a657c",
-        "SECURITY_AGENT_POLICIES_VERSION": "v0.10",
-        "MACOS_BUILD_VERSION": "6.29.0-rc.1",
+    "7.30.0-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.30.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.30.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.30.0-rc.1",
+        "JMXFETCH_VERSION": "0.44.2",
+        "JMXFETCH_HASH": "50ffa0f17549e85914ff664b1fc599475863a7cdd9f2ed1f62faea10d48f8ae8",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.13",
+        "MACOS_BUILD_VERSION": "7.30.0-rc.1",
         "WINDOWS_DDNPM_DRIVER": "release-signed",
         "WINDOWS_DDNPM_VERSION": "0.98.2.git.86.53d1ee4",
         "WINDOWS_DDNPM_SHASUM": "5d31cbf7aea921edd5ba34baf074e496749265a80468b65a034d3796558a909e"
     },
-    "7.29.0-rc.3": {
-        "INTEGRATIONS_CORE_VERSION": "7.29.0-rc.2",
-        "OMNIBUS_SOFTWARE_VERSION": "7.29.0-rc.1",
-        "OMNIBUS_RUBY_VERSION": "7.29.0-rc.1",
-        "JMXFETCH_VERSION": "0.44.1",
-        "JMXFETCH_HASH": "fd369da4fd24d18dabd7b33abcaac825d386b9558e70f1c621d797faec2a657c",
-        "SECURITY_AGENT_POLICIES_VERSION": "v0.10",
-        "MACOS_BUILD_VERSION": "7.29.0-rc.1",
+    "6.30.0-rc.1": {
+        "INTEGRATIONS_CORE_VERSION": "7.30.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.30.0-rc.1",
+        "OMNIBUS_RUBY_VERSION": "7.30.0-rc.1",
+        "JMXFETCH_VERSION": "0.44.2",
+        "JMXFETCH_HASH": "50ffa0f17549e85914ff664b1fc599475863a7cdd9f2ed1f62faea10d48f8ae8",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.13",
+        "MACOS_BUILD_VERSION": "6.30.0-rc.1",
         "WINDOWS_DDNPM_DRIVER": "release-signed",
         "WINDOWS_DDNPM_VERSION": "0.98.2.git.86.53d1ee4",
         "WINDOWS_DDNPM_SHASUM": "5d31cbf7aea921edd5ba34baf074e496749265a80468b65a034d3796558a909e"
     },
-    "6.29.0-rc.2": {
-        "INTEGRATIONS_CORE_VERSION": "7.29.0-rc.2",
-        "OMNIBUS_SOFTWARE_VERSION": "7.29.0-rc.1",
-        "OMNIBUS_RUBY_VERSION": "7.29.0-rc.1",
+    "6.29.0": {
+        "INTEGRATIONS_CORE_VERSION": "7.29.0",
+        "OMNIBUS_SOFTWARE_VERSION": "7.29.0",
+        "OMNIBUS_RUBY_VERSION": "7.29.0",
         "JMXFETCH_VERSION": "0.44.1",
         "JMXFETCH_HASH": "fd369da4fd24d18dabd7b33abcaac825d386b9558e70f1c621d797faec2a657c",
-        "SECURITY_AGENT_POLICIES_VERSION": "v0.10",
-        "MACOS_BUILD_VERSION": "6.29.0-rc.1",
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.12",
```

Comment:
In `6.29.0-rc.6` `SECURITY_AGENT_POLICIES_VERSION` is `v0.10`. Is `v0.12` correct?

Reply:
I believe so: https://github.com/DataDog/datadog-agent/blob/7.29.x/release.json#L67